index.html

```diff
@@ -131,13 +131,13 @@ <h2>About MySelf</h2>
         </p>
         <h3>LET'S CONNECT</h3>
       </div>
-      <div class="socialm">
-        <img src="/Asset/github.svg" alt="" />
-        <img src="/Asset/linkedin.svg" alt="" />
-        <img src="/Asset/angellist.svg" alt="" />
-        <img src="/Asset/twitter.svg" alt="" />
-        <img src="/Asset/meduim.svg" alt="" />
-      </div>
+      <ul class="socialm">
+        <li><img src="/Asset/github.svg" alt="" /></li>
+        <li><img src="/Asset/linkedin.svg" alt="" /></li>
+        <li><img src="/Asset/angellist.svg" alt="" /></li>
+        <li><img src="/Asset/twitter.svg" alt="" /></li>
+        <li><img src="/Asset/meduim.svg" alt="" /></li>
+      </ul>
```

Comment:
[OPTIONAL]
- [ ] Please always remember to add `alt` attribute with text describing the image to `img` tag. It enable screen readers to read the information about on-page images.